source/deployment/video-and-audio-calling.rst

```diff
@@ -37,10 +37,10 @@ BigBlueButton
 Skype for Business
 ~~~~~~~~~~~~~~~~~~~~
 
-- Start and join voice calls, video calls and use screensharing with your team members, developed by kosgrz.
+- Start and join voice calls, video calls and use screensharing with your team members, developed by Mattermost (Beta).
```

Comment:
```suggestion
- Start and join voice calls, video calls and use screensharing with your team members, developed by kosgrz, and maintained by Mattermost.
```

Reply:
Also, if you wanted to highlight that the plugin is beta for customer trust, that makes sense. Instead of adding it to the first line, we can add it to the title, `Sykpe for Business (Beta)`.

Reply:
Hi, @justinegeffen and @jasonblais 

Thank you for the feedback. Yes, I believe that retaining kosgrz as the original developer is a good idea.

And yes, the (Beta) on the title also works for me. All good.